### ISSUE TITLE
changes need in html and css

### ISSUE THREAD

**Issue by sanket2000sonavane** [incubateind/gdtc-Day2#88]
the size of icons in the facts section are not of equal size and shapes. can I make necessary changes in code @rexyagami ?

**Comment by rexyagami** [comment]
@sanket2000sonavane yes you can, there are some hints in the CSS file too you can find that it will make your work easier.  Good Luck

**Comment by krishnasinghdev** [comment]
Will you give me chance to do this .
